crmsh/bootstrap.py

```diff
@@ -679,7 +685,7 @@ def init_cluster_local():
         logger.warning("You should change the hacluster password to something more secure!")
 
     start_pacemaker(enable_flag=True)
-    wait_for_cluster()
+    wait_for_cluster(timeout_ms=120000)
```

Comment:
Since this number is used at many places, better to use a constant in constants.py

Reply:
Or, how about wait this time as a default value?